docs/thehive/setup/installation/3-node-cluster.md

```diff
@@ -135,37 +134,49 @@ GRANT ALL PERMISSIONS ON KEYSPACE thehive TO 'thehive';
 
 ### TheHive associated configuration
 
-Update the configuration of thehive accordingly in `/etc/thehive/application.conf` : 
-```
+Update the configuration of thehive accordingly in `/etc/thehive/application.conf` :
+
+```yaml title="/etc/thehive/application.conf"
 ## Database configuration
 db.janusgraph {
   storage {
     ## Cassandra configuration
     # More information at https://docs.janusgraph.org/basics/configuration-reference/#storagecql
-    backend: cql
-    hostname: ["<ip node 1>", "<ip node 2>", "<ip node 3>"]
+    backend = cql
+    hostname = ["<ip node 1>", "<ip node 2>", "<ip node 3>"]
     # Cassandra authentication (if configured)
-    username: "thehive"
-    password: "PASSWORD"
+    username = "thehive"
+    password = "PASSWORD"
     cql {
-      cluster-name: thp
-      keyspace: thehive
+      cluster-name = thp
+      keyspace = thehive
     }
   }
 ```
 
 ### Troubleshooting
 
-- > `InvalidRequest: code=2200 [Invalid query] message=”org.apache.cassandra.auth.CassandraRoleManager doesn’t support PASSWORD”.`
+!!! Example "Example of error message in /var/log/cassandra/ log files"
+
+    ```text
+    InvalidRequest: code=2200 [Invalid query] message=”org.apache.cassandra.auth.CassandraRoleManager doesn’t support PASSWORD”.`
+    ```
+
+    set the value `authenticator: PasswordAuthenticator` in `cassandra.yaml`
+
+    ---
 
-  set the value `authenticator: PasswordAuthenticator` in `cassandra.yaml`
+    ```
+    Caused by: java.util.concurrent.ExecutionException: com.datastax.driver.core.exceptions.UnauthorizedException: Unable to perform authorization of permissions: Unable to perform authorization of super-user permission: Cannot achieve consistency level LOCAL_ONE
+    ```
 
-- > `Caused by: java.util.concurrent.ExecutionException: com.datastax.driver.core.exceptions.UnauthorizedException: Unable to perform authorization of permissions: Unable to perform authorization of super-user permission: Cannot achieve consistency level LOCAL_ONE` 
+    Fix it by running following CQL command: 
 
-    ```sql
+    ``` sql
     ALTER KEYSPACE system_auth WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3 };
```

Comment:
After updating the replication factor, the command "nodetool repair" must be executed